gql/transport/transport.py

```diff
@@ -0,0 +1,20 @@
+import abc
```

Comment:
Maybe put this `transport/__init__.py` instead of creating a new file, so you can 'import Transport from gql.transport' or just 'import Transport from .' in local_schema and requests instead of 'import Transport from gql.transport.transport' which looks a bit ugly. And the submodules need the base class anyway.